centos_ci.sh

```diff
@@ -28,26 +31,26 @@ yum -y install \
   docker \
   kvm \
   qemu-kvm \
-  libvirt
+  libvirt \
+  python2-pip
+
+# Install 'requests' module used by tests/utils/minishift_latest_version.py
+pip install requests
```

Comment:
use python-request from the centos repo side. you can add it to yum -y install list.